components/tabs/src/schema.rs

```diff
@@ -34,10 +31,15 @@ const CREATE_META_TABLE_SQL: &str = "
 const CREATE_PENDING_REMOTE_DELETE_TABLE_SQL: &str = "
     CREATE TABLE IF NOT EXISTS pending_remote_tab_closures (
         id                      INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
I know this was already there, but might I suggest avoiding `AUTOINCREMENT`, if it's not too painful of a migration? It [has a cost](https://www.sqlite.org/autoinc.html), and it doesn't seem like we need its guarantees here.